esphome/components/deep_sleep/deep_sleep_component.cpp

```diff
@@ -29,9 +51,11 @@ void DeepSleepComponent::dump_config() {
   if (this->run_duration_.has_value()) {
     ESP_LOGCONFIG(TAG, "  Run Duration: %u ms", *this->run_duration_);
   }
-#ifdef USE_ESP32
-  if (wakeup_pin_ != nullptr) {
-    LOG_PIN("  Wakeup Pin: ", this->wakeup_pin_);
+#ifdef USEESP32
```

Comment:
```suggestion
#ifdef USE_ESP32
  if (wakeup_pin_ != nullptr)
    LOG_PIN("  Wakeup Pin: ", this->wakeup_pin_);
```
No reason to drop the logging of the wakeup pin.